### PR TITLE
Fix finspace clients

### PR DESCRIPTION
**Pull request by mullermp**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

1. To make sure we include your contribution in the release notes, please make sure to add description entry for your changes in the "unreleased changes" section of the `CHANGELOG.md` file (at corresponding gem). For the description entry, please make sure it lives in one line and starts with `Feature` or `Issue` in the correct format.

2. For generated code changes, please checkout below instructions first:
  https://github.com/aws/aws-sdk-ruby/blob/version-3/CONTRIBUTING.md

Thank you for your contribution!

### CODE REVIEW

File: gems/aws-sdk-finspace/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Add a `Content-Type` header to mitigate a service side issue.
```
Comment (joguSD) [inline gems/aws-sdk-finspace/CHANGELOG.md:4]: Let's note what the impacted services were.

Comment (mullermp) [reply gems/aws-sdk-finspace/CHANGELOG.md:4]: Each service has their own change log..

**Review by trevorrowe — APPROVED** [review]
The change looks good.

Curious question, have we tried to apply the content-type header to all rest-json services to see if this will work and could potentially be removed as a customization? This would need to be investigated.

**Comment by mullermp** [comment]
>Curious question, have we tried to apply the content-type header to all rest-json services to see if this will work and could potentially be removed as a customization? This would need to be investigated.

Unfortunately the rest json handler inherits from the rest handler, which applies to other protocols other than JSON.. I think it may be possible if we duplicate code.

**Pull request merged**